docs/source/status.rst

```diff
@@ -156,11 +156,11 @@ Flight RPC
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+
 | Flight RPC Transport                       | C++   | Java  | Go    | JavaScript | C#    | Rust  | Julia |
 +============================================+=======+=======+=======+============+=======+=======+=======+
-| gRPC_ transport (grpc:, grpc+tcp:)         | ✓     | ✓     | ✓     |            | ✓     |       |       |
+| gRPC_ transport (grpc:, grpc+tcp:)         | ✓     | ✓     | ✓     |            | ✓     | ✓     |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+
-| gRPC domain socket transport (grpc+unix:)  | ✓     | ✓     | ✓     |            | ✓     |       |       |
+| gRPC domain socket transport (grpc+unix:)  | ✓     | ✓     | ✓     |            | ✓     | ✓     |       |
 +--------------------------------------------+-------+-------+-------+------------+-------+-------+-------+
-| gRPC + TLS transport (grpc+tls:)           | ✓     | ✓     | ✓     |            | ✓     |       |       |
+| gRPC + TLS transport (grpc+tls:)           | ✓     | ✓     | ✓     |            | ✓     | ✓     |       |
```

Comment:
Arrow-flight uses tonic which supports [uds](https://github.com/hyperium/tonic/tree/master/examples/src/uds) and [tls](https://github.com/hyperium/tonic/tree/master/examples/src/tls)